### ISSUE TITLE
Progress info: support fractions

### ISSUE THREAD

**Issue by josteinaj** [daisy/pipeline-clientlib-java#12]
In addition to ranged (`[progress ... FROM-TO ...]`) and cumulative (`[progress ... INCREMENT ...]`) progress updates, also support fractions (`[progress ... POSITION/TOTAL ...]`).

Example:

``` xml
<px:message severity="DEBUG" message="[progress 175/6231]"/>
```

Probably also make cumulative progress updates work nicely with fractioned progress updates, so that this is possible:

``` xml
<px:message severity="DEBUG" message="[progress 0/1234]"/>

<px:message severity="DEBUG" message="[progress 1]"/>
<!-- progress is now 1/1234 -->

<px:message severity="DEBUG" message="[progress 1]"/>
<!-- progress is now 2/1234 -->
```

Fractioned progress info will make it easier to pass on progress info from within loops without having to do too many calculations in XPath/XProc:

``` xml
<px:message severity="DEBUG" message="[progress my:step 10 for-each.do-stuff]"/>
<p:for-each>
    <px:message severity="DEBUG">
        <p:with-option name="message" select="concat('[progress for-each.do-stuff ',p:iteration-position(),'/',p:iteration-size(),' for-each.do-stuff.iteration]')"/>
    </px:message>
    <px:message severity="DEBUG" message="[progress for-each.do-stuff.iteration 50 my:substep1]"/>
    <my:substep1/>
    <px:message severity="DEBUG" message="[progress for-each.do-stuff.iteration 50 my:substep2]"/>
    <my:substep2/>
</p:for-each>
```

**Comment by bertfrees** [comment]
@josteinaj Should this be closed?

**Comment by josteinaj** [comment]
Yes, seems it was fixed in #15.
